airflow-core/src/airflow/utils/__init__.py

```diff
@@ -46,5 +46,9 @@ def __getattr__(name: str):
     "xcom": {
         "XCOM_RETURN_KEY": "airflow.models.xcom.XCOM_RETURN_KEY",
     },
+    "decorators": {
+        "remove_task_decorator": "airflow.sdk.bases.decorator.remove_task_decorator",
```

Comment:
Since all of these are internal, you should move them to: `task-sdk/src/airflow/sdk/definitions/_internal/decorators.py` imo.

Reply:
Sorry, I mean to say public not internal

Reply:
Does it have a need to have it public? If there is no need to have it public, let's make it internal

Reply:
Agreed, maybe add a backcompat just to be sure or it becomes a breaking change?